OCP/tests/app/delivery_by_slack_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './app/delivery_by_slack'
+
+RSpec.describe DeliveryBySlack do
+  context 'when receiving a text' do
+    it 'delivers text by Slack' do
+      text = 'Olá, mundo!'
+      delivers_by_slack = puts 'Enviado por Slack.'
```

Comment:
@nataliakikuchi existe uma maneira de testar o puts de um maneira melhor... a @anaakisawa pode te explicar isso pois ela aprendeu isso esses dias...

Reply:
inclusive, o retorno do puts é `nil`. Então na prática, você tá fazendo ali embaixo é `expect(nil).to eq(nil)`.

![image](https://user-images.githubusercontent.com/2309096/100385379-2fcd0d00-3001-11eb-90a0-b75d7e1aab6a.png)

O "oi" é o que aparece na tela, o que tem depois do `=>` é o retorno (`nil`).